apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.ts

```diff
@@ -235,7 +235,10 @@ async function pipeCursorToArchive<T>(
   extractFiles: (row: T) => ArchiveFile[] | null,
 ) {
   const archive = archiver('zip');
-  const dirname = (res.locals.assessment_set.name + res.locals.assessment.number).replace(' ', '');
+  const dirname = (res.locals.assessment_set.name + res.locals.assessment.number).replaceAll(
```

Comment:
There's nothing stopping either an assessment set name of assessment number from each containing a space, or one containing multiple spaces.